### DISCUSSION TITLE
Confused

### DISCUSSION THREAD

**Discussion by userMak** [music-assistant/support#2769] [Q&A] [answered]
Hi
for some reason I have the main & the beta version in my system.
Do I need to have the both? What is the differences between them and how I could remove one of them if I need to? 

![image](https://github.com/user-attachments/assets/b84fa1c0-8623-4ea5-8a0d-79b4c50c5d9c)

**Comment by OzGav** [comment]
They are installed and removed through HA SETTINGS>>ADDONS

The beta has new functionality and bug fixes on test until it rolls out to stable.

No you don’t need both

**Reply by userMak** [reply]
Thanks. 
1. I am not sure which one is the default in my system. The beta is on addons and the stable in HACS right now. 

2. If I remove the beta for example since I have the the stable also the functionality of my system will remain or I will have to do/add/install something?

![image](https://github.com/user-attachments/assets/833a0edb-eb2b-4a70-be62-dfcb9302b4a9)

**Reply by OzGav** [reply]
Ok sorry I didn’t look closely enough before. Those two things are completely different. One is the integration and the other is the server. Review the docs for the difference. You need both if you want MA connected to HA

**Comment by userMak** [comment]
Sorry for the late reply. So I need both to run the beta version and I think this is the same for the stable version.
Can you please tell me how I can I add the stable version to remove the beta?
I tried to search it in HA addons but it is not available there

**Reply by OzGav** [reply, marked as answer]
You should be able to find it in the ADD ON STORE. Scroll down to the bottom if necessary.

**Comment by userMak** [comment]
Thanks!
